site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -16,13 +16,15 @@ authors:
 This document outlines a new proposal for identity federation: the Federated
 Credential Management API (FedCM).
 
-*  The [FedCM proposal](https://github.com/fedidcg/FedCM) has entered [public
+*  The [FedCM proposal](https://github.com/fedidcg/FedCM) is open for [public
    discussion](https://github.com/fedidcg/FedCM/issues).
-*  [FedCM's origin trial](/blog/fedcm-origin-trial) begins in Chrome 101 to 107
-   on Android. Chrome on desktop support starts in Chrome 103. Other browsers
-   don't support it yet.
-*  [The Privacy Sandbox timeline](http://privacysandbox.com/timeline) provides
-   implementation timings for FedCM and other Privacy Sandbox proposals.
+*  [FedCM origin trial](/blog/fedcm-origin-trial) ends in Chrome 107 and FedCM
+   is shipped in Chrome 108.
+*  FedCM isn't supported in other browsers yet, but Mozilla is [implementing a
```

Comment:
Here we have "Mozilla", "Firefox" and "WebKit". Could you use more consistent terms w.r.t. venders/browsers/engines?

Reply:
I have this fixed on my laptop. Will push the change once I get to it.

Reply:
This should have been addressed.